[INFRA] Transpose the entity table and link to text anchors describing each entity (#272)
bids-standard/bids-specification: src/02-common-principles.md

    due to different subject response or randomized nature of the stimuli). Run
    is a synonym of a data acquisition.

## File names consist of key-value pairs (entities)

> We define file names with a chain of key-value pairs, each of which we call

Reviewer: The "We" and "you" that show up a bit here feel off relative to most of the document. I think we generally use the passive voice or imperative mood. If it would be useful to have a native English speaker do a sweep through to uniformize the voice, I can do that.

Author: That'd be great :+1:

Reviewer: See sappelhoff#2.

an *entity*. Two prominent examples are the `subject` and `session` entities.

If we want to name a datafile that was collected in a certain `session` from
a certain `subject`, the name of that datafile will start with the following
string: `sub-<label>_ses-<label>`.

> Note that `sub-<sub-label>` correspondes to the `subject` entity because it has

Author: see https://github.com/bids-standard/bids-specification/pull/272#discussion_r307447751

but if both you and @yarikoptic agree I am happy to change it

> Note that `sub-<sub-label>` correspondes to the `subject` entity because it has

Reviewer: I think that the context makes it clear, so consistency with the rest of the document is more important here.

> the `sub-` "key" and`<sub-label>` "value", where `<sub-label>` would in a real

Reviewer 2: I think it should be just `<label>` instead of `<sub-label>` to stay consistent with the rest?

Author: I thought so too, but then decided against it to mark the point that the `<label>` is different between`subject` and `session` ... is that not helpful / helpful enough to break the consistency?

datafile correspond to a unique identifier of that subject, such as `01`. The
same  explanation holds for the `session` entity with its `ses-` key and its
`<ses-label>` value.

Finally, you can notice how different entities are separated by an underscore
(`_`).

With this system we can name datafiles in a human readable way:
`sub-01_task-rest_eeg.edf`

From the file name alone, a user can tell that the file contains resting state

> data from subject `01`. The *suffix* `_eeg` and the extension `.edf` are

Reviewer: Not sure about this one, but this puts "suffix" and "extension" on visual parity. Feel free to reject if you have a principled reason not to treat them the same.

```suggestion
data from subject `01`. The *suffix* `_eeg` and the *extension* `.edf` are
```

depending on the imaging modality and the data format and convey further
information regarding the file's contents.

> The list of all entities in BIDS and the order in which they SHOULD be

Reviewer: > SHOULD

Is this actually optional? I thought the ordering was semi-arbitrary but required, but I don't have a chapter and verse to cite.

Author: no you are right, this is required ... however, I don't think that the validator is completely strict on this at the moment
```suggestion
The list of all entities in BIDS and the order in which they MUST be
```

continuous recording file with the same naming scheme but different extensions.
The solutions will change from case to case and publicly available datasets will
be reviewed to include common data types in the future releases of the BIDS
specification.

## File name structure

> File names consist of chains of *entities*, or key-value pairs, a *suffix* and an

Reviewer: We were switching between plural and singular, and I think it works better to go consistently singular than consistently plural.

```suggestion
A file name consists of a chain of *entities*, or key-value pairs, a *suffix* and an
```
